step-functions-data-science-sdk/automate_model_retraining_workflow/automate_model_retraining_workflow.ipynb

```diff
@@ -0,0 +1,983 @@
+    "# You can use sagemaker.get_execution_role() if running inside sagemaker's notebook instance\n",
+    "sagemaker_execution_role = sagemaker.get_execution_role() #Replace with ARN if not in an AWS SageMaker notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create a Glue IAM Role\n",
+    "You need to create an IAM role so that you can create and execute an AWS Glue Job on your data in Amazon S3.\n",
+    "\n",
+    "1. Go to the [IAM console](https://console.aws.amazon.com/iam/).\n",
+    "2. Select **Roles** and then **Create role**.\n",
+    "3. Under **Choose the service that will use this role** select **Glue**.\n",
+    "4. Choose **Next** until you can enter a **Role name**.\n",
+    "5. Enter a name such as `AWS-Glue-S3-Bucket-Access` and then select **Create role**.\n",
+    "\n",
+    "Next, create and attach a policy to the role you created. The following steps attach a managed policy that provides Glue access to the specific S3 bucket holding your data.\n",
+    "\n",
+    "1. Under the **Permissions** tab, click **Attach policies** and then **Create policy**.\n",
+    "2. Enter the following in the **JSON** tab:\n",
+    "\n",
+    "```json\n",
+    "{\n",
+    "    \"Version\": \"2012-10-17\",\n",
+    "    \"Statement\": [\n",
+    "        {\n",
+    "            \"Sid\": \"ListObjectsInBucket\",\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": [\"s3:ListBucket\"],\n",
+    "            \"Resource\": [\"arn:aws:s3:::BUCKET-NAME\"]\n",
+    "        },\n",
+    "        {\n",
+    "            \"Sid\": \"AllObjectActions\",\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": \"s3:*Object\",\n",
+    "            \"Resource\": [\"arn:aws:s3:::BUCKET-NAME/*\"]\n",
+    "        }\n",
+    "    ]\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "3. Run the next cell (below) to retrieve the specific **S3 bucket name** that we will grant permissions to."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "session = sagemaker.Session()\n",
+    "bucket = session.default_bucket()\n",
+    "print(bucket)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "4. Copy the output of the above cell and replace the **two occurances** of **BUCKET-NAME** in the JSON text that you entered.\n",
+    "5. Choose **Review policy** and give the policy a name such as `S3BucketAccessPolicy`.\n",
+    "6. Choose **Create policy**.\n",
+    "7. Select **Roles**, then search for and select your `AWS-Glue-S3-Bucket-Access` role.\n",
+    "8. Under the **Permissions** tab, click **Attach policies**.\n",
+    "9. Search for your newly created `S3BucketAccessPolicy` policy and select the check box next to it.\n",
+    "10. Choose **Attach policy**. You will then be redirected to the details page for the role.\n",
```

Comment:
[Minor] I don't see anything wrong with the json, but bullet points 10 and 11 don't show up properly in the notebook preview.
Can you briefly verify that these look good on your end?

Reply:
When I open the file in ReviewNM, I don't see bullet points 10 and 11 either. However when I open the notebook in a SageMaker notebook I see them. 
![NotebookNumbering](https://user-images.githubusercontent.com/44590366/74291567-136a1900-4cea-11ea-9e70-7d5b2e91d788.PNG)

Reply:
Perfect. Thanks for the verification!